API_OVERVIEW.md

```diff
@@ -76,24 +76,24 @@ for the v2 API change process.
 
 Unless otherwise stated, the APIs with the same names as v1 APIs have a similar role.
 
-* [Cluster Discovery Service (CDS)](api/cds.proto).
-* [Endpoint Discovery Service (EDS)](api/eds.proto). This has the same role as SDS in the [v1 API](https://www.envoyproxy.io/docs/envoy/latest/configuration/cluster_manager/sds_api.html),
+* [Cluster Discovery Service (CDS)](envoy/service/discovery/v2/cds.proto).
```

Comment:
Unfortunately, this is about to change back in #438, stay tuned. Thanks for jumping in and fixing though.

Reply:
Thanks for checking. Got it.

I also found some broken links in sphinx docs, so once the refactoring has been completed, we should run a script to fix all the links.

Closing this temporal fix and wait the on-going refactoring, thanks.